sdk/appconfiguration/app-configuration/test/package.spec.ts

```diff
@@ -0,0 +1,9 @@
+const packageJson = require("../package.json");
```

Comment:
Did you check if this test runs as expected in CI?
The last time we tried this, it required us to use the json plugin for rollup which resulted in its own set of issues

Some links from the past
https://github.com/Azure/azure-sdk-for-js/pull/3017#issuecomment-496749006
#3224
#3302

It could be possible that the issue before was due to the package.json being required in the code where we set the user agent string and not so much as in the test code.

But sync with @ramya0820 for details.

Reply:
They pass _but_ I'm not doing a browser bundle (at the moment) so that's probably expected.

With that said - no reason to make things more difficult for future-me if we decide to make a browser bundle so I've changed it over to @ramya0820 's superior method instead. :)

Reply:
Yes, this specifically needed a different way of importing the file for addressing a browserification issue. It was a tricky issue, so it's good we are safeguarding early on.